modules/ppcp-webhooks/src/Handler/VaultPaymentTokenCreated.php

```diff
@@ -149,4 +155,29 @@ public function handle_request( WP_REST_Request $request ): WP_REST_Response {
 
 		return $this->success_response();
 	}
+
+	/**
+	 * Returns WC customer id from PayPal customer id.
+	 *
+	 * @param string $customer_id The customer ID from PayPal.
+	 * @return int
+	 */
+	private function wc_customer_id_from( string $customer_id ): int {
+		$customers = get_users(
```

Comment:
There is a phpcs warning about possibly slow query. If there is no way to do it differently, then should `phpcs:ignore` it.